app/controllers/schools/add_participants/details_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Schools::AddParticipants
+  class DetailsController < BaseController
+    def show; end
+  end
```

Comment:
Is it a good idea to inherit from a controller that has a `start` action? Will that action get inherited? 

Same for `TypesController`.

Reply:
Good point. It is definitely inherited, but without the route it will stay unused. As this is the first draft of this feature I think I'd get it as it is and will clean it up a bit later (most likely separate start / finalize, both inheiriting completley abstract base?)

Reply:
Uhh, this reads like a multi-page creation flow with a shared session store. Is that what it is?

If so, can we have a single controller for it? It will mean we can keep the convention of 'the only place that changes the session with a key is a controller that defines this key' - and well, `TypesController` and `DetailsController` are pretty tiny as is.

Reply:
@kicferk1 - Yup, actually got annoyed by this as well yesterday and this is now corrected in the next PR.

Reply:
Alright, sooo, what exactly should I check in this PR? 😂 

What's the split between this and next one exactly?

Reply:
@kicferk1 Yup, that's exactly my question - how do we divide the comments between multiple PRs? After all, the feature is still in progress. I wonder if maybe it will be better to have one larger PR, but with well defined, coherent commits so it can be reviewed commit by commit instead?

Reply:
I think in the future, I would prefer a single PR that tries to be pretty minimal in how it attempts to implement the feature. If it means we need to add a ticket of tech debt here or there, with a todo comment in code, so be it.